template/.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,79 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: "34 5 * * 0"
+  workflow_dispatch:
+  push:
+    branches: ["main"]
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  analysis:
+    name: Scorecard analysis
+    runs-on: ubuntu-latest
+    # `publish_results: true` only works when run from the default branch. conditional can be removed if disabled.
+    if: github.event.repository.default_branch == github.ref_name || github.event_name == 'pull_request'
```

Comment:
do we need something like `github.repository == '{{ github_url | split("/")[-2:].join("/") }}'`?

Reply:
Why? So that it doesn't run on forks?

I literally copied the version here we use in `pixi-pack`.

Reply:
> So that it doesn't run on forks?

yes. we should maybe also adjust it in pixi-pack as well?

Reply:
I have disabled it although I see no downside of running it on forks (GitHub Actions is nowadays anyways disabled by default on forks)